nvme.c

```diff
@@ -4308,6 +4346,9 @@ static int device_self_test(int argc, char **argv, struct command *cmd, struct p
 		fprintf(stderr, "Device self-test: %s\n", nvme_strerror(errno));
 
 close_dev:
+	if (err == -EINTR)
+		abort_self_test(&args);
```

Comment:
I am wondering if this is something what the user expects? Well, I see both arguments, a) we just want to stop the foreground wait but not the background operation and b) we always want to stop everything with ctrl-c. There is properly for both situation an good reason why a certain behavior is preferred.

I think we just go with your proposal and address any feedback (if there is any).

Reply:
I have just checked the dst command implementation again so as far as I checked the err `-EINTR' is only set by the foreground wait so seems no problem and concern.

Reply:
Thanks for testing. As said, I think we just see what people think of it.